### ISSUE TITLE
Null collection returned

### ISSUE THREAD

**Issue by jaszfalvitamas** [GeniusesOfSymfony/PubSubRouterBundle#21]
Hi!

When I clear the Symfony cache via command line, I get a critical error, see below. I did not encounter any issues on the website yet.

```sh
CRITICAL  [php] Uncaught Error: Return value of Gos\Bundle\PubSubRouterBundle\Router\Router::getCollection() must be an instance of Gos\Bundle\PubSubRouterBundle\Router\RouteCollection, null returned
[
  "exception" => TypeError {
    #message: "Return value of Gos\Bundle\PubSubRouterBundle\Router\Router::getCollection() must be an instance of Gos\Bundle\PubSubRouterBundle\Router\RouteCollection, null returned"
    #code: 0
    #file: "./vendor/gos/pubsub-router-bundle/src/Router/Router.php"
    #line: 165
    trace: {
      ./vendor/gos/pubsub-router-bundle/src/Router/Router.php:165 { …}
      ./vendor/gos/pubsub-router-bundle/src/Router/Router.php:272 { …}
      ./vendor/gos/pubsub-router-bundle/src/Router/Router.php:247 { …}
      ./vendor/symfony/config/ConfigCacheFactory.php:42 { …}
      ./vendor/gos/pubsub-router-bundle/src/Router/Router.php:255 { …}
      ./vendor/gos/pubsub-router-bundle/src/Router/Router.php:174 { …}
      ./vendor/gos/pubsub-router-bundle/src/CacheWarmer/RouterCacheWarmer.php:36 { …}
      ./vendor/symfony/http-kernel/CacheWarmer/CacheWarmerAggregate.php:95 { …}
      ./vendor/symfony/framework-bundle/Command/CacheClearCommand.php:196 { …}
      ./vendor/symfony/framework-bundle/Command/CacheClearCommand.php:129 { …}
      ./vendor/symfony/console/Command/Command.php:255 { …}
      ./vendor/symfony/console/Application.php:943 { …}
      ./vendor/symfony/framework-bundle/Console/Application.php:97 { …}
      ./vendor/symfony/console/Application.php:265 { …}
      ./vendor/symfony/framework-bundle/Console/Application.php:83 { …}
      ./vendor/symfony/console/Application.php:141 { …}
      ./bin/console:33 {
        › $application = new Application($kernel);\r
        › $application->run($input);\r
        ›
      }
    }
  }
]
```

**Comment by mbabker** [comment]
Can you paste in your config for this bundle?  And if you have GosWebSocketBundle installed, that one as well.  I think I see how this can happen, but a config I can reproduce this with would help a lot.

**Comment by jaszfalvitamas** [comment]
Only the basic configuration:
gos_web_socket:
    server:
        port: 8080        # The port the socket server will listen on
        host: 127.0.0.1   # The host ip to bind to

**Comment by mbabker** [comment]
Fixed and released in 1.5.0 and 2.1.0.
